### DISCUSSION TITLE
How can I add line numbers to text in a PDF?

### DISCUSSION THREAD

**Discussion by asimonis** [quarto-dev/quarto-cli#3875] [Q&A]
Reviewers often use line numbers to refer to edits and suggestions on a manuscript. I've found options to [insert line numbers for code](https://quarto.org/docs/reference/formats/pdf.html#numbering), but there's no mention of how to number lines of text. 

I've found the [Latex package 'lineno'](https://ctan.org/pkg/lineno?lang=en), but am not sure how to use it when working with Quarto through RStudio. 

Thanks for any tips or suggestions!

**Comment by cscheid** [comment]
It's possible that quarto has an option to do so explicitly, but I couldn't find it either. As a minimal do-it-yourself solution, you can use this in your frontmatter to add LaTeX content to the header of your document:

```
format: 
  pdf:
    include-in-header: 
      text: '\usepackage{lineno}\linenumbers'
```

**Reply by asimonis** [reply]
Brilliant! That worked for me. Thanks Carlos!
